src/System.Management.Automation/engine/lang/parserutils.cs

```diff
@@ -282,6 +282,7 @@ static ParserOps()
         private static readonly string[] s_chars = new string[255];
```

Comment:
@IISResetMe, your last commit had 2 failures in `PowerShell-CI-macos`
Tab completion bug fix.Issue#1350 - '1 -a<tab>' should work
```powershell
Expected a collection with size 2, but got collection with size 4 @(System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult).
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Host/TabCompletion/BugFix.Tests.ps1: line 19
19:         $result.CompletionMatches | Should -HaveCount 2
```
TabCompletion.Completion on 'comma', 'redirection' and 'minus' tokens.Test complete the minus token to operators
```powershell
Expected strings to be the same, but they were different.
String lengths are both 4.
Strings differ at index 2.
Expected: '-and'
But was:  '-all'
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 1000
1000:             $res.CompletionMatches[0].CompletionText | Should -BeExactly '-and'
```